lingui.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('@lingui/conf').LinguiConfig} */
+module.exports = {
+	locales: ["en", "fr"],
```

Comment:
The fr locale is just being used as a dummy locale for testing. It can be removed without issue and lingui will still function without it. I can also change the language to one that you know you want to start translating to.

Reply:
I think I am inclined to remove this before we merge. It's useful to see how it all hangs together but I think I'll need a bit of time to figure out what translations we want to put into place and who will do that work.